src/main/java/org/graylog/plugins/outputs/neo4j/Neo4jOutput.java

```diff
@@ -167,7 +129,9 @@ public ConfigurationRequest getRequestedConfiguration() {
                             CK_NEO4J_QUERY, "Cypher query",
                             "MERGE (source:HOST { address: '${source}' })\n" +
                                     "MERGE (user_id:USER { user_id: '${user_id}'})\nMERGE (source)-[:CONNECT]->(user_id)",
-                            "Query will be executed on every log message. Use template substitutions to access message fields: ${took_ms}",
+                            "Query will be executed on every log message.\n" +
+                                    "HTTP Mode: Use template substitutions to access message fields: ${took_ms}\n" +
+                                    "Bolt Mode: Use curly brackets only to access message fields: {took_ms}",
```

Comment:
Sounds good to me, maybe it's better to put this in a second PR so that we can keep this one here clear.